Barotrauma/BarotraumaClient/ClientSource/GUI/GUITextBox.cs

```diff
@@ -617,10 +617,15 @@ public void ReceiveCommandInput(char command)
                 case '\b' when !Readonly: //backspace
                 {
                     using var _ = new TextPosPreservation(this);
-                    if (PlayerInput.KeyDown(Keys.LeftControl) || PlayerInput.KeyDown(Keys.RightControl))
+                    if (PlayerInput.IsCtrlDown())
                     {
-                        SetText(string.Empty, false);
-                        CaretIndex = Text.Length;
+                        var removeTo = FindStopSymbolLeft(Text, CaretIndex);
+                        var len = CaretIndex - removeTo;
+                        var newText = Text.Remove(removeTo, CaretIndex - removeTo);
+                        SetText(newText, false);
```

Comment:
Should be `SetText(newText, store: true)`, or undo/redo memento won't register this.